frontend/src/pages/chartTest.js

```diff
@@ -1,107 +1,157 @@
-import React, { useState } from 'react';
-import {Button, Grid} from '@material-ui/core';
-import Radar from './chart/chart';
-
+import React, { useState } from "react";
+import { Button, Grid } from "@material-ui/core";
+import Radar from "./chart/chart";
 
 function RadarTest(props) {
-
-    const [input, setInput] = useState(0);
-    const [testInput1, settestInput1] = useState(0);
-    const [testInput2, settestInput2] = useState(0);
-    const [testInput3, settestInput3] = useState(0);
-
-
-    const scenario_scoring_matrix = {
-        Tim: {
-            issue1: 1,
-            issue2: 4,
-            issue3: 2
-        },
-        Song: {
-            issue1: 0,
-            issue2: 2,
-            issue3: 3
-        }
-
-    };
-
+  const [input, setInput] = useState(0);
+  const [testInput1, setTestInput1] = useState(0);
+  const [testInput2, setTestInput2] = useState(0);
+  const [testInput3, setTestInput3] = useState(0);
+
+  const scenario_scoring_matrix = {
+    Tim: {
+      issue1: 1,
+      issue2: 4,
+      issue3: 2,
+    },
+    Song: {
+      issue1: 0,
+      issue2: 2,
+      issue3: 3,
+    },
+  };
+  function Total_EmphasisF(scenario_scoring_matrix) {
```

Comment:
Is this consistent of how everything across the app will be named? The mix of camel case and snake case seems a but off to me. I think picking camel case (totalEmphasisFunction) or snake case (total_emphasis_function) would be good (generally react projects use camel but its up to you)